core/src/main/java/greencity/controller/SuperAdminController.java

```diff
@@ -144,55 +148,64 @@ public ResponseEntity<GetTariffServiceDto> editTariffService(
      *
      * @param dto  {@link CreateServiceDto}
      * @param uuid {@link String} - user uuid.
-     * @return {@link CreateServiceDto}
-     * @author Vadym Makitra.
+     * @return {@link ServiceDto}
+     * @author Vadym Makitra
+     * @author Julia Seti.
      */
 
     @ApiOperation(value = "Add new service")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK),
-        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = CreateServiceDto.class),
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = ServiceDto.class),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
         @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
-        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
     })
     @PreAuthorize("@preAuthorizer.hasAuthority('CONTROL_SERVICE', authentication)")
     @PostMapping("/createService")
-    public ResponseEntity<CreateServiceDto> createServices(
+    public ResponseEntity<ServiceDto> createServices(
```

Comment:
I see that we could return next responses:

- `201` `CREATED`  response if we successfully create the service
- `401` `UNAUTHORIZED`  if the user is not authorized
- `403` `FORBIDDEN`  if the user does not have the required authority
- `400` `BAD_REQUEST` if the service already exists in the DB or the request does not contain a valid service to create

But I do not see the flow when we return `200` `OK` and `404` `NOT_FOUND`. Could you please provide examples and add tests for these response codes?

Reply:
https://github.com/ita-social-projects/GreenCityUBS/blob/52813d75715cabc3378e3969814cd67831bfcb5b/core/src/test/java/greencity/controller/SuperAdminControllerTest.java#L179

https://github.com/ita-social-projects/GreenCityUBS/blob/52813d75715cabc3378e3969814cd67831bfcb5b/core/src/test/java/greencity/controller/SuperAdminControllerTest.java#L204

200 - already deleted